content/en/docs/_index.md

```diff
@@ -1,20 +1,18 @@
 
 ---
-title: "Tekton Documentation"
+title: "Welcome to Tekton"
 linkTitle: "Documentation"
 weight: 20
 menu:
   main:
     weight: 20
 ---
 
-Tekton is a cloud native continuous integration and delivery (CI/CD)
-solution. It allows developers to build, test, and deploy across cloud
-providers and on-premise systems.
-
-### What's next
-
-To get started, read the contents below.
+Tekton is a cloud-native solution for building CI/CD pipelines. It consists of Tekton Pipelines, which provides
+the building blocks, and of supporting components, such as Tekton CLI and Tekton Catalog, that make Tekton a complete ecosystem.  
+Tekton is part of the [CD Foundation](https://cd.foundation/), a [Linux Foundation](https://www.linuxfoundation.org/projects/) project. For more information, see the [Overview of Tekton](/docs/overview). 
+ 
+![Pipeline](/docs/concepts/concept-tasks-pipelines.png)
```

Comment:
I would remove the image. It applies to Pipelines only and we already have a separate overview for Pipelines that explains how it works. Also, the link is broken right now. The rest looks good.

Reply:
Hi, @sergetron thanks a lot for reviewed. As requestion of "add a diagram or two"  in #195, I added pipeline image here. Because pipeline is the core function in Tekton.  If you have any other image about Tekton more suitable here, would you please share it? Or NO image here is OK? @afrittoli 
 @AlanGreene

Reply:
That image belongs in the Tekton Pipelines docs, not on the main Tekton landing page. It's too much technical detail for a landing page.

Reply:
Thank you @sergetron for your helpful advice! This is @kobaji working with @cclemonzy .
Honestly our team could not prepare for the image which describe the entire Tekton project.

On the other hand, this PR is related to the issue #195 and it requires to add some diagrams regarding Tekton, therefore our team are considering if we should add the diagram or image.

@afrittoli 
Could we just improve the documentation landing page without adding a diagram or two?
I apologize for this inconvenience and not being able to merge sooner.

Reply:
I think it's best to:
1. Remove the current image.
2. Merge this PR.
3. Add a new, better image in a new PR.

Reply:
Thank you @sergetron for your kindness!!

@cclemonzy 
Could you remove the current image?

Reply:
I have removed the pipeline image just now.  @sergetron @kobaji  Appreciate your timely help.